modules/migration-migrating-applications-api.adoc

```diff
@@ -44,6 +44,22 @@ You can create multiple `MigMigration` CRs for a single `MigPlan` CR for the fol
 * The _version_ of the installed {mtc-full} Operator must be the same on all clusters.
```

Comment:
Out of scope for this PR but to match wording with the other prereqs, considers:

```suggestion
* You must use the same version of the installed {mtc-full} Operator on all clusters.
```

Also, I don't think that "version" needs to be italicized. (Our [OCP doc guidance](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference) is to use underscores for emphasis sparingly.)

Reply:
OK, but this is a BIG prerequisite that people seem to overlook easily. Fine.

Reply:
Ah, I see. Your call. :)